lib/compute/service.js

```diff
@@ -0,0 +1,300 @@
+     * object as well.
+     *
+     * @param {options=} options - Configuration object.
+     * @param {boolean} options.autoCreate - Automatically create the object if
+     *     it does not exist. Default: `false`
+     *
+     * @example
+     * service.get(function(err, service, apiResponse) {
+     *   // `service` is a Service object.
+     * });
+     */
+    get: true,
+
+    /**
+     * Get the metadata of this backend service.
+     *
+     * @resource [BackendService Resource]{@link https://cloud.google.com/compute/docs/reference/v1/backendServices#resource}
+     * @resource [BackendService: get API Documentation]{@link https://cloud.google.com/compute/docs/reference/v1/backendServices/get}
+     *
+     * @param {function=} callback - The callback function.
+     * @param {?error} callback.err - An error returned while making this
+     *     request.
+     * @param {object} callback.metadata - The service's metadata.
+     * @param {object} callback.apiResponse - The full API response.
+     *
+     * @example
+     * service.getMetadata(function(err, metadata, apiResponse) {});
+     */
+    getMetadata: true
+  };
+
+  ServiceObject.call(this, {
+    parent: compute,
+    baseUrl: '/global/backendServices',
+    id: name,
+    createMethod: compute.createService.bind(compute),
+    methods: methods
+  });
+
+  this.compute = compute;
+  this.name = name;
+}
+
+nodeutil.inherits(Service, ServiceObject);
+
+/**
+ * Delete the backend service.
+ *
+ * @resource [BackendServices: delete API Documentation]{@link https://cloud.google.com/compute/docs/reference/v1/backendServices/delete}
+ *
+ * @param {function=} callback - The callback function.
+ * @param {?error} callback.err - An error returned while making this request.
+ * @param {module:compute/operation} callback.operation - An operation object
+ *     that can be used to check the status of the request.
+ * @param {object} callback.apiResponse - The full API response.
+ *
+ * @example
+ * service.delete(function(err, operation, apiResponse) {
+ *   // `operation` is an Operation object that can be used to check the status
+ *   // of the request.
+ * });
+ */
+Service.prototype.delete = function(callback) {
+  var compute = this.compute;
+
+  callback = callback || util.noop;
+
+  ServiceObject.prototype.delete.call(this, function(err, resp) {
+    if (err) {
+      callback(err, null, resp);
+      return;
+    }
+
+    var operation = compute.operation(resp.name);
+    operation.metadata = resp;
+
+    callback(null, operation, resp);
+  });
+};
+
+/**
+ * Get the most recent health check results.
+ *
+ * @resource [BackendServices: getHealth API Documentation]{@link https://cloud.google.com/compute/docs/reference/latest/backendServices/getHealth}
+ *
+ * @param {string|object} group - The fully-qualified URL of an Instance Group
+ *     resource.
+ * @param {string} - group.name - The name of the Instance Group resource.
```

Comment:
The first hyphen causes a formatting error - http://stephenplusplus.github.io/gcloud-node/#/docs/master/compute/service?method=getHealth